cmd/forwarder/fixer.go

```diff
@@ -45,8 +45,15 @@ func fix(r io.Reader, remoteAddr string, logplexDrainToken string) ([]byte, erro
 		messageWriter.WriteString(" [origin ip=\"")
 		messageWriter.WriteString(remoteAddr)
 		messageWriter.WriteString("\"]")
-
 		b := lp.Bytes()
+
+		var replace bool
+
+		if authUser != "" && bytes.HasSuffix(b, []byte("\n")) {
```

Comment:
@freeformz I'm not 100% sure that the `HasSuffix` is necessary here, if the `replace` handling isn't... see comment below.

Reply:
put this data in *front* of the log message call `messageWriter.WriteString(" logg_iss_user="+authUser)` here.

Reply:
That was my first, thought. However, it looks like, based on the test output, that it supports structured data in the in coming log line. In which case, putting it here would break that up and make the data in the log line no longer valid structured data.

Reply:
To be a bit more verbose on that, if you look at this example from the current state tests
```
138 <13>1 2013-06-07T13:17:49.468822+00:00 host heroku web.7 - [origin ip=\"1.2.3.4\"][60607e20-f12d-483e-aa89-ffaf954e7527] log_iss_user=ingest
```

Adding it here would result in 
```
138 <13>1 2013-06-07T13:17:49.468822+00:00 host heroku web.7 - [origin ip=\"1.2.3.4\"] log_iss_user=ingest [60607e20-f12d-483e-aa89-ffaf954e7527]
```

Which I don't think is what we want.